src/sentry/suspect_resolutions/resolved_in_active_release.py

```diff
@@ -0,0 +1,38 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models import Deploy, Group, GroupStatus, Release
+
+
+def is_resolved_issue_within_active_release(issue_id: int) -> bool:
```

Comment:
Same suggestion as before, does it make sense to pass in the specific Group instead of `issue_id`?